models/skillImpactsModel.js

```diff
@@ -0,0 +1,17 @@
+const db = require('../database/dbConfig');
+
+
+async function findSkills(campaign_id) {
+    const skills = await db('skilled_impact_requests')
+        .where({campaign_id})
+        .join('project_goals', 'project_goals.skilled_impact_request_id', 'skilled_impact_requests.id')
+        .select(
+            '*'
+        )
+        .first();
```

Comment:
Shouldn't adding this `first()` call should only return the first result and not all skilled impact requests matching the given `campaign_id`?